connector/keystone/keystone.go

```diff
@@ -215,25 +233,49 @@ func (p *conn) getAdminToken(ctx context.Context) (string, error) {
 }
 
 func (p *conn) checkIfUserExists(ctx context.Context, userID string, token string) (bool, error) {
+	user, err := p.getUser(ctx, userID, token)
+	if err != nil {
+		return false, err
+	}
+	if user == nil {
+		return false, err
+	}
+	return true, nil
+}
```

Comment:
Nit: looks like this function is almost equivalent to
```go
func (p *conn) checkIfUserExists(ctx context.Context, userID string, token string) (bool, error) {
	user, err := p.getUser(ctx, userID, token)
		
	return user == nil, err
} 
```

Reply:
Thanks for your Feedback!

Definitely... changing to `return user != nil, err` (`!=` instead of `==`).